src/odemis/gui/comp/stream_panel.py

```diff
@@ -808,6 +808,10 @@ def _add_side_label(self, label_text, tooltip=None):
                           flag=wx.ALL | wx.ALIGN_CENTER_VERTICAL, border=5)
         return lbl_ctrl
 
+    def Destroy(self):
```

Comment:
I am not sure it works, is it called somewhere?

Reply:
To clarify, `.Destroy()` is part of the standard API in wx objects. So it's automatically called by wxPython when the component is destroyed. This method overrides the standard version. 
@sneepstefan , I think to do it "properly" you should call `super().Destroy()` from here.